src/main/java/org/jabref/gui/preferences/journals/AbbreviationViewModel.java

```diff
@@ -94,4 +94,18 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(getName(), isPseudoAbbreviation());
     }
+
+    public boolean containsCaseIndependent(String s) {
+        s = s.toLowerCase();
+        if (this.abbreviation.get().toLowerCase().contains(s)) {
```

Comment:
You can simplify this: 
```suggestion
       return this.abbreviation.get().toLowerCase(Locale.ROOT).contains(s)) || this.name.get().toLowerCase(Locale.ROOT).contains(s)) || this.shortestUniqueAbbreviation.get().toLowerCase(Locale.ROOT).contains(s))  {
```

Reply:
Is this easier to read though?

Reply:
Looking over it again I agree with you and changed it.